pyproject.toml

```diff
@@ -22,7 +22,7 @@ dependencies = [
     "ansys-api-fluent>=0.3.35",
     "ansys-platform-instancemanagement~=1.1",
     "ansys-tools-filetransfer>=0.1,<0.3",
-    "ansys-units>=0.3.3,<0.5",
+    "ansys-units>=0.3.3,<0.6",
```

Comment:
@hpohekar What us the purpose of the upper limit?

Reply:
@seanpearsonuk To allow installation of dev version.

Reply:
@hpohekar I mean what is the purpose of having any upper limit?

Reply:
I would remove it if possible - I just did the less intrusive modification

Reply:
@RobPasMue Actually we can keep it as `"ansys-units>=0.4.0"` . Earlier it was in same format without an upper limit.

Reply:
Yes, please, @hpohekar.

Reply:
> I would remove it if possible - I just did the less intrusive modification

@RobPasMue That was my assumption, too, and thank you.

Reply:
Done. Thank you.

Reply:
As per the @mkundu1's suggestion we are keeping major version as an upper limit. Thank you.